slider/collection.py

```diff
@@ -0,0 +1,98 @@
+# TODO: not use replay private functions
+from .replay import _consume_int, _consume_string
+
+
+class CollectionDB:
+    """An osu! ``collection.db`` file.
+
+    Parameters
+    ----------
+    version : int
+        Version number (e.g. 20150203)
+    num_collections : int
+        Number of collections
+    collections : list[Collection]
+        List of :class:`~slider.collection.Collection` s
+    """
+    def __init__(self, version, num_collections, collections):
+        self.version = version
+        self.num_collections = num_collections
+        self.collections = collections
```

Comment:
Do we need the `num_collections` or can we use `len(collections)` here?

Reply:
Same as https://github.com/llllllllll/slider/pull/50#discussion_r301883855